mdoc/Mono.Documentation/Updater/Formatters/AttributeFormatters/AttributeFormatter.cs

```diff
@@ -101,8 +102,12 @@ public virtual bool TryGetAttributeString(CustomAttribute attribute, out string
                         (from namedArg in attribute.Properties
                          select new { Type = namedArg.Argument.Type, Name = namedArg.Name, Value = namedArg.Argument.Value }))
                 .OrderBy(v => v.Name);
-            foreach (var d in namedArgs)
-                fields.Add(MakeNamedArgumentString(d.Name, MakeAttributesValueString(d.Value, d.Type)));
+            foreach (var d in namedArgs) {
+                string namedArgument = MakeNamedArgumentString(d.Name, MakeAttributesValueString(d.Value, d.Type));
+                if (Language == Consts.CppWinRt && namedArgument.StartsWith("typeof("))
```

Comment:
Same comment here ... you should override `MakeNamedArgumentString` in the c++ formatter, and put the logic there